src/FullImageSegmentationAnnotator/index.story.js

```diff
@@ -70,7 +109,7 @@ storiesOf("FullImageSegmentationAnnotator.Basic", module)
             })),
           },
         ]}
-        regionClsList={["bg", "orange" , "hand"]}
+        regionClsList={["orange", "bg", "hand"]}
```

Comment:
this was originally correct, now incorrect

Reply:
"bg" should be first

Reply:
You right, I don't know why I changed like this. It must be an accident.
https://github.com/waoai/react-image-annotate/pull/63/commits/b777e5ff2f270acfb4cf11551223856a6cfad0af